src/content/doc-surrealdb/installation/running/docker.mdx

```diff
@@ -38,40 +38,42 @@ In order to persist data when the Docker instance is restarted or shutdown, spec
 
 ```bash
 mkdir mydata # Create a directory to store the database, owned by the current user
-docker run --rm --pull always -p 8000:8000 --user $(id -u) -v $(pwd)/mydata:/mydata surrealdb/surrealdb:latest start file:/mydata/mydatabase.db
+docker run --rm --pull always -p 8000:8000 --user $(id -u) -v $(pwd)/mydata:/mydata surrealdb/surrealdb:latest start rocksdb:/mydata/mydatabase.db
 ```
 
 The default logging level for the database server is `info`, resulting in any informational logs to be output to the standard output. To control the logging verbosity, specify the *`--log`* argument. The following command starts the database with `trace` level logging, resulting in most logs being output to the terminal.
 
 ```bash
 mkdir mydata # Create a directory to store the database, owned by the current user
-docker run --rm --pull always -p 8000:8000 --user $(id -u) -v $(pwd)/mydata:/mydata surrealdb/surrealdb:latest start --log trace file:/mydata/mydatabase.db
+docker run --rm --pull always -p 8000:8000 --user $(id -u) -v $(pwd)/mydata:/mydata surrealdb/surrealdb:latest start --log trace rocksdb:/mydata/mydatabase.db
 ```
 
 ### Configuring authentication
 
-In order to keep SurrealDB secure, configure your initial root-level user by setting the *`--user`* and *`--pass`* command-line arguments. Authentication is required by default.
+Authentication is enabled by default since version 2.0, using the `--unauthenticated` flag to opt out. In versions of SurrealDB before 2.0, authentication was disabled by default and required an `--auth` flag to enable.
```

Comment:
```suggestion
Authentication is enabled by default since version `2.0`, using the `--unauthenticated` flag to opt out. In versions of SurrealDB before 2.0, authentication was disabled by default and required an `--auth` flag to enable.
```